src/makeCalculator.js

```diff
@@ -37,7 +37,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+    operate(operation, val) {
```

Comment:
не сокращай имена переменных
```suggestion
    operate(operation, value) {
```